spring-cloud-generator/generate-one.sh

```diff
@@ -112,7 +112,7 @@ add_module_to_pom () {
     if [[ ${3:-1} -eq 1 ]]; then
       # we need to remove any previously existing entries so the entry points to the
       # right $monorepo_commitish's readme
-      sed -i "/$starter_artifactid/d" README.md
+      sed -i "/com.google.cloud:$starter_artifactid/d" README.md
```

Comment:
Removing entries by matching on `$starter_artifactid` alone also removes an unrelated line that contains an example in the dependency xml snippet above.

Reply:
@emmileaf, I realized that since we are now [using the empty template](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/b256dc5e0bbdc9694ee3a2a19c2b5d91b30361ef/spring-cloud-generator/reset-previews-folder.sh#L7-L9) with an empty readme, we can maybe just remove this line

Reply:
@diegomarquezp Ah you're right, thanks for the suggestion! I'll quickly test and remove this line.